experimental/PlaneCurve/AffinePlaneCurve.jl

```diff
@@ -344,10 +530,26 @@ end
     arithmetic_genus(C::AffinePlaneCurve)
 
 Return the arithmetic genus of the projective closure of `C`.
+
+# Example
+```repl
```

Comment:
At the beginning I worked from PR #470, but I got a lot of errors, I tried to rebase on a more version of Oscar, but since PR #466 was not there, the genus was not working. I will have to check if this repl and several other ones can be changed into a jldoctest...